build-system/server/variable-substitution-tester.js

```diff
@@ -0,0 +1,147 @@
+/**
```

Comment:
Why is this file called `-tester`?

Reply:
Called it tester because it can be used to test our analytics requests easily. 

Open to alternatives that are better fitting

Reply:
I think `variable-substitution.js` would work fine here.